$NetBSD: patch-Configure,v 1.10 2020/07/10 10:04:54 hauke Exp $

OpenSSL mandates that you need to re-create dependencies for
unorthodox combinations of build options. These days, the compiler
will create dependency information. On older platforms (e.g. pre-v9
Darwin) Configure will fall back to makedepend(8), which hasn't kept up
well with compilers' options.

Instead of failing the build on makedepend(8) malfunction, have
make(1) ignore its return value - which used to be the default for
previous OpenSSL versions.

--- Configure.orig	2023-03-14 12:59:07.000000000 +0000
+++ Configure
@@ -1574,7 +1574,7 @@ if (!$disabled{makedepend}) {
     } else {
         # In all other cases, we look for 'makedepend', and set the
         # makedep_scheme value if we found it.
-        $config{makedepcmd} = which('makedepend');
+        $config{makedepcmd} = '-' . which('makedepend');
         $config{makedep_scheme} = 'makedepend' if $config{makedepcmd};
     }
 
